specification/vmware/Microsoft.AVS/routes.tsp

```diff
@@ -38,9 +38,15 @@ interface Locations {
 interface PrivateClouds {
   #suppress "@azure-tools/typespec-azure-core/no-operation-id" "Can not change existing operationId."
   @operationId("PrivateClouds_List")
-  list is ArmResourceListByParent<PrivateCloud>;
+  list is ArmResourceListByParent<
```

Comment:
will the naming fix be covered in 0.57.0? if yes, we will need to regen after typespec is released for this right?

Reply:
For this release, we likely would generate from Swagger. And operationId in Swagger is specified on above `@operationId("PrivateClouds_List")` line.

We are renaming these List in another PR (in client.tsp) https://github.com/Azure/azure-rest-api-specs/pull/29308
But as we know, the problem is how to rename resource group...